rtb/piano_di_progetto_ver0.2.0.typ

```diff
@@ -220,12 +227,88 @@ Le attività che il gruppo si è preoccupato di concludere durante queste due se
 
 Si evidenzia come difetto la possibilità di lavorare su una stessa _#rifGlossario("branch")_ a più membri in quanto questa causerebbe problemi di conflitti tra i file modificati.
 
-Gli analisti evidenziano le difficoltà e i dubbi relativa alla comprensione dei casi d'uso del prodotto richiesto dal proponente, che hanno però trovato risposta dalla riunione di fine primo _sprint_ da parte del proponente e che verranno approfonditi in quello successivo. Incentivano inoltre a lavorare in maniera più collaborativa alla fase di analisi e non limitarsi semplicemente all'esecuzione delle attività di analisi in maniera parallela.
+Gli analisti evidenziano le difficoltà e i dubbi relativi alla comprensione dei casi d'uso del prodotto richiesto dal proponente, che hanno però trovato risposta dalla riunione di fine primo _sprint_ da parte del proponente e che verranno approfonditi in quello successivo. Incentivano inoltre a lavorare in maniera più collaborativa alla fase di analisi e non limitarsi semplicemente all'esecuzione delle attività di analisi in maniera parallela.
 
 I verificatori suggeriscono infine di incaricare più di una persona alla verifica dei documenti ad alto contenuto, quali possono essere le Norme di Progetto e il Piano di Progetto, al fine di garantire una verifica ottimale.
 
 
+#pagebreak()
+
+=== Sprint 2
+- *Inizio*: 28/11/2024
+- *Fine*: 13/12/2024
+- *Rischi attesi*: #link(<RO-2>)[RO-2], #link(<RO-3>)[RO-3], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+//Quello che è stato fatto o quello che è da fare ?
+Le attività che il _team_ si è prefissato di portare a termine durante il secondo _sprint_ sono le seguenti:
+- Redazione verbale esterno ed interno del 27/11/2024.
+- Creare una prima versione del simulatore dati.
+  - È stato deciso di dedicare la prima settimana dello _sprint_ allo studio e approfondimento delle tecnologie: #rifGlossario("Docker"), #rifGlossario("Apache Kafka"), #rifGlossario("Apache Nifi") e #rifGlossario("PostgresSQL").
+  - La seconda settimana sarà invece riservata all'implementazione concreta del simulatore e del _database_ dove verranno salvati i dati generati, grazie all'utilizzo delle tecnologie studiate.
+- Avanzamento nel documento Analisi dei Requisiti, in particolare: 
+  - Migliorando i vecchi casi d'uso dopo aver risolto alcuni dubbi in seguito al confronto con l'azienda.
+  - Scrivendo nuovi casi d'uso.
```

Comment:
- scriverei "migliorare" e "scrivere" al posto del gerundio sempre per il discorso di non collegare le frasi in un elenco puntato